changelog/next/bug-fixes/5108--chronicle.md

```diff
@@ -0,0 +1,2 @@
+The default max request size has been changed from `4MB` to `1MB` for
+`to_google_secops` to align with the API.
```

Comment:
Active voice please.

Reply:
Also, is this really a bug fix and not a change?

Reply:
Yes, it is a bug fix. The API docs state a limit of 4MiB, but in reality the limit turned out to be 1MiB, making the operator unusable when you went over the limit.

Reply:
This is worthwhile putting in a giant callout.